### PR TITLE
Add version check for Katello when running the db seed.

### PR DESCRIPTION
**Pull request by zjhuntin**

### CODE REVIEW

File: puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/test/test_katello.sh [hunk 1]
```diff
@@ -73,9 +73,15 @@ bundle exec rake db:migrate
...
 # Katello-specific tests
 bundle exec rake jenkins:katello TESTOPTS="-v"
+
 # Run the DB seeds to verify they work
-bundle exec rake db:drop db:create db:migrate
-bundle exec rake db:seed
+# Don't run DB seeds if the version of katello is less than 3.1
+VERSION=$(grep -Po "(\d+\.)+\d+" $PLUGIN_ROOT + '/lib/katello/version.rb')
+
+if [ $(echo ${VERSION}$'\n' '3.1.0' | sort --version-sort --reverse | head -n1) != ${VERSION} ]; then
```
Comment (ekohl) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/test/test_katello.sh]: Is the space before ```3.1.0``` intentional?

Comment (zjhuntin) [reply puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/test/test_katello.sh]: It was not, thanks for catching that!

**Comment by ekohl** [comment]
Thanks!

**Comment by ekohl** [comment]
Looks like this fails. Ruby-style concats it appears
```
23:40:09 ++ grep -Po '(\d+\.)+\d+' /var/lib/workspace/workspace/test_develop_pr_katello/database/postgresql/ruby/2.2/slave/fast/plugin + /lib/katello/version.rb
23:40:09 grep: /var/lib/workspace/workspace/test_develop_pr_katello/database/postgresql/ruby/2.2/slave/fast/plugin: Is a directory
23:40:09 grep: +: No such file or directory
23:40:09 grep: /lib/katello/version.rb: No such file or directory
23:40:09 + VERSION=
```

**Comment by iNecas** [comment]
Here you go https://github.com/theforeman/foreman-infra/pull/393

**Pull request merged**